atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/sweep/queue/TargetedSweepTest.java

```diff
@@ -194,6 +194,18 @@ public void sweepDeletesAllIfLatestCommittedTombstoneInThorough() {
         assertNoEntryForCellInKvs(TABLE_THOR, TEST_CELL);
     }
 
+    @Test
+    public void sweepThrowsAwayWritesForDroppedTables() {
+        useOneSweepQueueShard();
+        writeInTransactionAndGetStartTimestamp(SINGLE_WRITE);
+        Long startTimestamp = writeInTransactionAndGetStartTimestamp(SINGLE_WRITE);
+
+        keyValueService.dropTable(TABLE_CONS);
```

Comment:
fun story - you can't put this test in TargetedSweeperTest because the kvs in that one has automaticallyCreateTables = true, so this is useless.